uber/static/angular-apps/hotel/app.js

```diff
@@ -43,10 +43,10 @@ angular.module('hotel', ['ngRoute', 'magfest'])
     })
     .config(function($routeProvider){
         $routeProvider
```

Comment:
We used to have 3 different things in the ``uber/static/lib`` directory:
* third party modules we use
* our own static library files we wrote which are used in more than one place
* our Angular apps for various parts of the site

The first I deleted since they're no longer necessary.  The second and third I moved elsewhere to help with the refactor; since the ``static/lib`` directory no longer exists, it made it easy to grep for and update all of the places where it was used before.  Our directory structure is also now a little flatter, which I personally prefer.